content/documentation/overview/index.adoc

```diff
@@ -115,6 +117,92 @@ Kiali will highlight configuration errors.
 image::istio-yaml-validation.png[Demo Website]
 {nbsp} +
 
+==== Validations performed
+
+In this section, there are listed all the validations that Kiali perform for all the Istio Configs. Most of these validations are extra to the existent ones performed by Galley (excepting those marked as deprecated).
```

Comment:
How about:
"This section lists all the validations that Kiali performs on all Istio configurations. Most of these validations are done in addition to/on top of the existing ones performed by Istio's Galley component (except those marked as deprecated)"